tox.ini

```diff
@@ -47,9 +47,10 @@ require-code = true
 [testenv]
 commands =
     pipenv install
+    sh {toxinidir}/scripts/create_postgresDB.sh
     {toxinidir}/superset/bin/superset db upgrade
     nosetests tests/load_examples_test.py
-    nosetests -e load_examples_test {posargs}
+    nosetests -e load_examples_test {posargs} --with-xunit --cover-xml
 deps =
```

Comment:
can we move db create to jenkins stages like--
pretest
test
posttest

Reply:
we will take it as a different task to improve jenkinsfile